tests/FakeItEasy.Tests.Approval/ApiApproval.cs

```diff
@@ -15,28 +15,38 @@ public class ApiApproval
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public void ApproveApi()
+        public void ApproveApi45()
         {
-            Approvals.Verify(PublicApiGenerator.GetPublicApi(typeof(A).Assembly));
+            ApproveApi("net45");

```

Comment:
No, it's fine; it's better to do it the same way for all assemblies, when possible. But the project reference to FakeItEasy should be removed from this project, since it's no longer necessary.